### ISSUE TITLE
Generate data set for Tree LSTM

### ISSUE THREAD

**Issue by data-compute-geek** [zackchase/mxnet-the-straight-dope#334]
Is the code for generating the data set "tree_lstm_dataset-3d85a6c4.cPickle" (used in chapter09) available?
I would like to generate the parse tree for my own text, and build a similar model.

Thanks a lot.

**Comment by zackchase** [comment]
@szha please help! Thanks :)

**Comment by szha** [comment]
The script can be found in https://github.com/apache/incubator-mxnet/tree/master/example/gluon/tree_lstm. Check [the shell script](https://github.com/apache/incubator-mxnet/blob/master/example/gluon/tree_lstm/fetch_and_preprocess.sh) for downloading and parsing trees, and [main.py](https://github.com/apache/incubator-mxnet/blob/master/example/gluon/tree_lstm/main.py) for loading and pickling.

**Comment by data-compute-geek** [comment]
Hi Zack & Sheng,
Thank you both very much for the quick responses. This is really helpful!
-Sean

**Comment by data-compute-geek** [comment]
Hi Zha Sheng,

Thank you for the help, and now I am able to train tree lstms for my
application. However the model seems to overfit the training data since the
error on the dev set is noticeably larger than the training error.
I wondering if you have implemented dropout or regularization to avoid
overfitting, or when this might be available if you plan to do so.

Thank you very much.
Xianlong
